### ISSUE TITLE
DOC101: allow whitelisting specific arguments from being required for the docstring

### ISSUE THREAD

**Issue by ssbarnea** [jsh9/pydoclint#142]
Especially when it comes to tests, there is a series of pytest fixtures that do not really need any documentations. A simple example would be the code below:

```python
@pytest.fixture(autouse=True)
def _no_color(monkeypatch: pytest.MonkeyPatch) -> None:
    """Disable coloring output.
    ...
```

I do think that if we could define a list of keywords / args that would not need to be documented, we could sort this problem.

Update with other attributes/arguments that we do not want to document:

```
__metaclass__
__next__
```

**Comment by jsh9** [comment]
Hi @ssbarnea , the risk of excluding certain arguments is ambiguity and communication overhead.

For example, you don't want `arg1` and `arg2` to ever be checked by pydoclint.  Then what should pydoclint do if it finds documentations of `arg1 `and `arg2` in the docstring, especially, if the documentation of `arg1` and `arg2` is partially correct (e.g., correct name, incorrect type hint)?  Should pydoclint report a violation?

If pydoclint doesn't need to complain, future maintainers of the repo may get super confused (because they see that some args can "get away" with being wrong).  They'd either need to ask the initial authors, or come and ask me.  All this, in my opinion, is communication overhead that slows a team down.

If, on the other hand, you'd want to exclude certain functions/classes/methods from being checked by pydoclint, it is supported via flake8.  You can put `# noqa: DOC` at the line of function/classes definition.

**Comment by jsh9** [comment]
I'm closing this issue now.  If there are additional things that I missed, please feel free to add a comment here or open a new issue.
